test/test_linalg.py

```diff
@@ -6142,6 +6142,26 @@ def genf_int_float(x, y, use_transpose, non_contig_type):
         torch._int_mm(a_int8, b_int8, out=c_int32_result)
         self.assertEqual(c_int32_result.float(), torch.mm(a_float, b_float))
 
+    @unittest.skipIf(IS_WINDOWS, "Skipped on Windows!")
+    @unittest.skipIf(IS_FBCODE and IS_REMOTE_GPU, "cublas runtime error")
+    @onlyNativeDeviceTypes
+    def test__convert_weight_to_int4pack(self, device):
```

Comment:
maybe you can link the issue in comment as well

Reply:
Done.